dkron/ui.go

```diff
@@ -81,7 +81,7 @@ func (h *HTTPTransport) UI(r *gin.RouterGroup) {
 				ln = l.Name
 			}
 			ctx.HTML(http.StatusOK, "index.html", gin.H{
-				"DKRON_API_URL":          fmt.Sprintf("/%s", apiPathPrefix),
+				"DKRON_API_URL":          fmt.Sprintf("../%s", apiPathPrefix),
```

Comment:
Is this working serving the API from a root path?

Reply:
It should as UI is served from /ui but I’ll double check

Reply:
Can you confirm?

Reply:
sorry, I forgot but yes, accessing directly (on root path) still works as expected